chat-ui/src/DevCenterChatbot.tsx

```diff
@@ -361,20 +367,35 @@ function ChatbotModal({
 const MAX_INPUT_CHARACTERS = 300;
 interface InputBarProps extends LGInputBarProps {
   inputBarValue: string;
-  hasError: boolean;
+  inputBarHasError: boolean;
+  conversationError: string | undefined;
 }
 
 const InputBar = (props: InputBarProps) => {
-  const { inputBarValue, hasError, ...LGInputBarProps } = props;
+  const {
+    inputBarValue,
+    inputBarHasError,
+    conversationError,
+    ...LGInputBarProps
+  } = props;
   const { darkMode } = useDarkMode();
 
+  if (conversationError)
+    return (
+      <div className={styles.chatbot_input_area}>
+        <ErrorBanner darkMode={darkMode} message={conversationError} />
+      </div>
+    );
```

Comment:
I don't think conversation errors are the concern of the input bar, especially given that we use it here to short circuit rendering. In the docs `Chatbot` component we conditionally render the InputBar based on conversation error state rather than have the InputBar handle the conditional internally.

Could we refactor here to use that pattern instead? It'll make it easier to combine this all into shared components.